Explorer/Assets/DCL/AvatarRendering/Wearables/Systems/ResolveWearableByPointerSystem.cs

```diff
@@ -146,8 +146,11 @@ private void FinalizeWearableDTO([Data] bool defaultWearablesResolved, in Entity
                 {
                     foreach (WearableDTO assetEntity in promiseResult.Asset.Value)
                     {
-                        //TODO: Download Thumbnail
                         wearableCatalog.TryGetWearable(assetEntity.metadata.id, out IWearable component);
+                        var wearableThumbnailComponent = new WearableThumbnailComponent(component);
+
+                        //TODO Handle thumbnails on a ResolutionLevel after character preview outline is merged
+                        World.Create(wearableThumbnailComponent, PartitionComponent.TOP_PRIORITY);
```

Comment:
I believe this entity should contain Partition component as we should inherit it. Otherwise we don't take into account the original priority the wearable was requested with